packages/backend/src/ee/index.ts

```diff
@@ -247,6 +248,7 @@ export async function getEnterpriseAppArguments(): Promise<EnterpriseAppArgument
                     userAttributesModel: models.getUserAttributesModel(),
                     searchModel: models.getSearchModel(),
                     spaceModel: models.getSpaceModel(),
+                    spaceService: repository.getSpaceService(),
```

Comment:
not ideal, but our AI services depend on other services at the moment

Reply:
McpService -> McpClient? A discussion for another day